handlers/line/line.go

```diff
@@ -32,6 +32,14 @@ var (
 	signatureHeader = "X-Line-Signature"
 )
 
+// see https://core.telegram.org/bots/api#sending-files
+var mediaSupport = map[handlers.MediaType]handlers.MediaTypeSupport{
+	handlers.MediaTypeImage:       {Types: []string{"image/jpe", "image/jpg", "image/jpeg", "image/png"}, MaxBytes: 10 * 1024 * 1024},
```

Comment:
I don't think I've ever seen a browser upload something as `image/jpe`.. I'm inclined to say if there's a chance that can happen we should have rapidpro correct `image/jpe` to `image/jpeg` rather than have every channel handler have to worry about `image/jpe` vs `image/jpg` vs `image/jpeg`

Reply:
Updated